prowler/compliance/aws/cis_2.0_aws.json

```diff
@@ -643,9 +643,9 @@
       ],
       "Attributes": [
         {
-          "Section": "2.4 Relational Database Service (RDS)",
+          "Section": "2.4 Elastic File System (EFS)",
           "Profile": "Level 1",
-          "AssessmentStatus": "Manual",
+          "AssessmentStatus": "Automated",
```

Comment:
```suggestion
          "AssessmentStatus": "Manual",
```
Same here

Reply:
Afraid I have to disagree. This is an automated check and not manual as per prowler\providers\aws\services\efs\efs_encryption_at_rest_enabled\efs_encryption_at_rest_enabled.py

I also ran this against my AWS account and confirm its an automated check. This is not a manual check.

Any new EFS file systems are created with Encryption enabled and cant be turned off. You can only move to a customer managed key too.